Description: Temporarily disable rkyv
 Nothing in Debian currently depends on chrno+rkyv, so temporarily disable to
 allow upgrading rkyv to 0.8.
Author: Blair Noctis <ncts@debian.org>
Forwarded: not-needed
Last-Update: 2025-01-23
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -289,3 +289,3 @@
 
-[dependencies.rkyv]
+[disabled.dependencies.rkyv]
 version = "0.7.41"
@@ -379,12 +379,12 @@
 rand-no-std = ["rand-package"]
-rkyv-safe-deser = [
-    "rkyv-serialize",
-    "rkyv/validation",
-]
-rkyv-serialize = [
-    "rkyv-serialize-no-std",
-    "rkyv/std",
-    "rkyv/validation",
-]
-rkyv-serialize-no-std = ["rkyv/size_32"]
+#rkyv-safe-deser = [
+#    "rkyv-serialize",
+#    "rkyv/validation",
+#]
+#rkyv-serialize = [
+#    "rkyv-serialize-no-std",
+#    "rkyv/std",
+#    "rkyv/validation",
+#]
+#rkyv-serialize-no-std = ["rkyv/size_32"]
 serde-serialize = [
